package-system/zlib/install_zlib_linux.sh

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+OUT_PATH=$TARGET_INSTALL_ROOT
+SRC_PATH=temp/src
+BLD_PATH=temp/build
+
+mkdir -p $OUT_PATH
+
+cp -f $SRC_PATH/LICENSE $OUT_PATH/
+mkdir -p $OUT_PATH/lib
```

Comment:
same error check should go to the .sh scripts as well.

Reply:
Added these too.

Reply:
note that I tested by intentionally making a misspelling so that it fails.  (Temporariily!)